### PR TITLE
Module 1: Middle section My Portfolio Website

### PR DESCRIPTION
**Pull request by alvinlouis29**
##The main objective of this project is to understand how to parse a Figma design to create a UI using Flexbox and CSS Grid to ##place elements in the page.

## In this Branch I have added my recent works and about section as indicated in the Figma design

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -55,5 +55,146 @@ <h1> Hey there. I’m Alvin <br>
...
+  </div>
+  <div class="content">
+  <h1>Multi-post Stories <br>Gain + Glory</h1>
+  <ul class="category">
+  <li><a href="#">Ruby on Rails</a></li>
+  <li><a href="#">Css</a></li>
+  <li><a href="#">Javascript</a></li>
+  <li><a href="#">Html</a></li>
+  </ul>
+   <div class="open"><a href="#">See Project</a></div>
```
Comment (OLIPLICHE) [inline index.html]: - [x] The button `See project` follows the button styling: Active, Hover, and Pressed (see design).
- [ ] 
image

File: index.html [hunk 2]
```diff
@@ -55,5 +55,146 @@ <h1> Hey there. I’m Alvin <br>
   <button class="btn arrow"> <i class="fa fa-angle-down"> </i></button>
   </section>
   </section>
+
+  <section class="works_section">
+  <div class="recent_works_title">
+  <h1>My Recent Works</h1>
+  </div>
+  <div class="cards">
```
Comment (OLIPLICHE) [inline index.html]: - [x] Please Follow the[ Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A988) and add what is missing in your design.   

- [x] **Original Design**

see1

- [x] **Your Design**

see2

File: index.html [hunk 3]
```diff
@@ -55,5 +55,146 @@ <h1> Hey there. I’m Alvin <br>
...
+<section class="about">
+  <h1>About</h1>
+  <div class="about_info">
+  <p> I can help you build a product , 
+  feature or website Look through some 
+  of my work and experience! If you like 
+  what you see and have a project you need coded, 
+  don’t hestiate to contact me.</p>
+  </div>
+  <div class="resume"><a href="#">Get My Resume</a></div>
```
Comment (OLIPLICHE) [inline index.html]: - [x] Please The button `Get my resume` follows the button styling: Active, Hover, and Pressed (see design)

image

File: index.html [hunk 4]
```diff
@@ -55,5 +55,146 @@ <h1> Hey there. I’m Alvin <br>
...
+  <li><a href="#">Javascript</a></li>
+  <li><a href="#">Ruby</a></li>
+  <li><a href="#">Html</a></li>
+  <li><a href="#">Css</a></li>
+  </ul>
+  </div>
+  </li>
+  <li class="weight">
+  <div class="featured frame-works"></div>
+  <h1>Frameworks</h1>
```
Comment (OLIPLICHE) [inline index.html]: - [x] Please the technology categories ("Languages", "Frameworks" and "Skills") are in a `<ul>`, each of them in a `<li>`

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @alvinlouis29 , 

Good job so far!
There are some issues that you still need to work on to get your project approved, but you are almost there!

## Highlights Good Points: 👍
✔️ No linters error.
✔️ Github flow used properly.
✔️ Pull Request has a descriptive title telling you what changes are inside.


## Required Changes ♻️

- [x] Please Remove the `Live Demo` section in your ReadMe. It is not important for now.

- [ ]  CHECK THE COMMENTS UNDER THE REVIEW⬇..

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**You can also consider:**
- Pull Request should have a short summary describing introduced changes in more detail.
- Please beautify your HTML code before pushing it to GitHub, I strongly recommend that.  Otherwise, your work will be considered **unprofessional**.

- [x] **Professional**:

- [x] id1

- [x] **Not Professional**

- [x] id2



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by smunozmo — APPROVED** [review]
Hi @alvinlouis29, 👋 

Good job making all the required changes, your project is complete!  There is nothing else to say other than... it's time to merge it! 🚀 

Very nice work, congratulations! 🎉 

### APPROVED! 👏 👏 👏 




Cheers and Happy coding! 🤓 

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**